src/webgl/texture.js

```diff
@@ -666,6 +687,27 @@ export default class Texture extends Resource {
   _setParameter(pname, param) {
     this.gl.bindTexture(this.target, this.handle);
 
+    // NOTE: Apply NPOT workaround
+    if (this._isNPOTWorkaroundApplicable()) {
+      const supportedMinFilters = [GL.LINEAR, GL.NEAREST];
```

Comment:
Define a constant at top of file
`const NPOT_MIN_FILTERS`
you could make it an object, then you don't need to use indexOf.